.github/workflows/interop.yml

```diff
@@ -89,32 +89,10 @@ jobs:
 
       # ---------------------------------------------------------------------------------------
 
-#      - name: Test interoperability
-#        run: |
-#          ./target/debug/interop_client&
-#          ./mlspp/cmd/interop/mlspp_client -live 12345&
-#
-#          cd mls-implementations/interop
-#          # TODO(#1238):
-#          # * Add `commit.json` as soon as group context extensions proposals are supported.
-#          # Note: It's also possible to remove GCE proposals by hand from `commit.json`.
-#          #       But let's not do this in CI for now and hope that it isn't needed anymore soon.
-#          for scenario in {welcome_join.json,external_join.json,application.json};
-#          do
-#            echo Running configs/$scenario
-#            errors=$(./test-runner/test-runner -fail-fast -client localhost:50051 -client localhost:12345 -config=configs/$scenario | grep error | wc -l)
-#            if [ "$errors" = "0" ];
-#            then
-#              echo "Success";
-#            else
-#              echo "Failed";
-#              exit 1;
-#            fi
-#          done
-
-      - name: Test interoperability (OpenMLS only)
+      - name: Test interoperability
         run: |
           ./target/debug/interop_client&
+          ./mlspp/cmd/interop/mlspp_client -live 12345&
```

Comment:
Are these changes down here on purpose?

Reply:
This PR really just `git revert`s the PR that disabled MLS++ interop, so there shouldn't be any changes.